src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenNullCheckedParameterTests.cs

```diff
@@ -1399,12 +1398,26 @@ .maxstack  3
         public void TestNullCheckedEnumeratorInLocalFunction()
         {
             var source = @"
+using System;
 using System.Collections.Generic;
+
+Iterators.Use();
+
 class Iterators
 {
-    void Use()
+    public static void Use()
     {
         IEnumerator<char> e = GetChars(""hello"");
+        Console.Write(1);
+
+        try
+        {
+            GetChars(null!);
+        }
+        catch
```

Comment:
>catch [](http://example.com/codeflow?start=8&length=5)

Consider catching `ArgumentNullException` explicitly, here and in the other new tests that catch exceptions.